src/chain_spec.rs

```diff
@@ -167,7 +167,7 @@ macro_rules! checkpoint {
 // (primary_account_id, service provider did, target did, expiry time of CDD claim i.e 10 days is ms)
 type Identity = (
     AccountId,
-    IdentityId,
+    Vec<IdentityId>,
```

Comment:
Who do we need this to be a vector? We are only passing 1 cdd provider everywhere.

Reply:
Because we do pass 2 providers in runtime tests and without this modification tests won't work.

Reply:
Could we use a `Smallvec<[_; 2]>` instead or better yet cfg-switch on Vec/vs. not?